plugin.audio.cbcradio/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Please include language info via the `<language>` tag with the 2-letter language code.

Reply:
done